telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -53,7 +62,34 @@ async def callback(update: Update, context: CallbackContext)
 
     """
 
-    __slots__ = ()
+    __slots__ = (
+        "_chat_ids",
+        "_usernames",
+    )
+
+    def __init__(
+        self,
+        callback: HandlerCallback[Update, CCT, RT],
+        chat_id: SCT[int] = None,
+        username: SCT[str] = None,
+        block: DVInput[bool] = DEFAULT_TRUE,
+    ):
+        super().__init__(callback, block=block)
+
+        if chat_id is not None:
+            if isinstance(chat_id, int):
+                chat_id = frozenset({chat_id})
+            else:
+                chat_id = frozenset(chat_id)
+
+        if username is not None:
```

Comment:
usernames should be sanitized to add/drop a leading `@` IMO. Please also add a corresponding unit test that makes sure that both versions are accepted.

Reply:
Dropp the @ please, telegram doesn't send it with the @ either.